examples/backends/vllm/launch/disagg_tracing.sh

```
@@ -2,7 +2,10 @@
 # SPDX-FileCopyrightText: Copyright (c) 2025-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 set -e
-trap 'echo Cleaning up...; kill 0' EXIT
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../common/launch_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)